src/PerformanceTests/Tests/Tools/TestEnvironment.cs

```diff
@@ -153,7 +164,7 @@ static void Clone(string src, string dst)
                 if (equalTimestamps) return;
                 File.Delete(dst);
             }
-            if (!SymbolicLink.Create(src, dst)) File.Copy(src, dst);
+            File.Copy(src, dst);
```

Comment:
Not using symlinks will require large amounts of storage if you are going to run all tests!!

Reply:
@ramonsmits but the number of tests & permutations which are capable of running on linux/.net core is fairly small.

Reply:
@timbussmann If you run all tests on V5,V6,V7, and V7 dotnet core, that is alot of data. I agree that is you only do a subset then it isn't really an issue.

Reply:
@danielmarbach Is that ` ReparsePointUtility` available on net452 too?

Reply:
I would be in favor of just cloning that method, although I don't know how it would perform but probably fast enough compared to an API call.

Reply:
Also, this is part of the test runner, do we want that to be able to run on dotnet core?

Reply:
> Also, this is part of the test runner, do we want that to be able to run on dotnet core?

if we want to run the tests on linux we kinda have to unless you want to involve a lot of manual steps.

Reply:
A lot of manual steps? Probably just the creation of a script that does a recursive search on NServiceBus?.dll/exe and then execute it or maybe I'm simpllfying it too much but it we can use the test runner that actually can generate the permutations that would surely be really nice.

Reply:
without dotnet core support you would have to generate the permutation output on a windows machine before copying files over to platform you want to run as you already said, that makes for a very annoying testing process. 
Additionally you would have to make sure that all the dependencies (nuget packages) are available, which might be tricky, given that you cannot use dotnet restore (as you don't have a csproj available).